classes/Tools.php

```diff
@@ -2907,7 +2907,7 @@ public static function generateRobotsFile($executeHook = false)
         }
 
         // Sitemap
-        if (file_exists($sitemap_file) && filesize($sitemap_file)) {
+        if (file_exists($sitemap_file) && filesize($sitemap_file) && isset($_SERVER['SERVER_NAME'])) {
```

Comment:
this will not work with CLI

Reply:
I think this was the goal of this PR but what about using `Tools::getDomains()` instead of `$_SERVER['SERVER_NAME']` so it can works in both situations?

Reply:
I also don't understand why the sitemap is not configured for all domains :sweat_smile:

Reply:
🆙  @Progi1984 I think the above suggestion from @PierreRambaud makes sense 😉